cmd/telemetry-gateway/internal/server/samsm2m/samsm2m.go

```diff
@@ -14,21 +13,27 @@ import (
 
 	"github.com/sourcegraph/log"
 
+	"github.com/sourcegraph/sourcegraph-accounts-sdk-go/scopes"
+
+	sams "github.com/sourcegraph/sourcegraph-accounts-sdk-go"
 	"github.com/sourcegraph/sourcegraph/internal/authbearer"
-	"github.com/sourcegraph/sourcegraph/internal/sams"
 	"github.com/sourcegraph/sourcegraph/lib/errors"
 )
 
-const requiredSamsScope = "telemetry_gateway::events::write"
+var requiredSamsScope = scopes.ToScope("telemetry_gateway", "events", "write")
```

Comment:
If you're up to go one step further 😁 

```suggestion
var requiredSamsScope = scopes.ToScope(scopes.ServiceTelemetryGateway, "events", scopes.ActionWrite)
```

Reply:
🖖